### PR TITLE
inplemented the view for the index with bootstrap and css and fixed an issue with gerardo in seeds.rb file

### PR DESCRIPTION
**Pull request by KarimIssa1**
let me know if i need to fix anything.
Image 2024-04-27 at 17 39 49_c6e8d828

### CODE REVIEW

**Comment by lauradumi** [comment]
I would maybe add them into a container so that there's some space one the sides of the page 
I would also add more spacing between the title and the actual cards

**Comment by KarimIssa1** [comment]
i did like laura suggested and put the h1 tag into a container and added some margin bottom 
updated locations view

**Comment by KarimIssa1** [comment]
added spaces ont he sides of the cards to make it a little bit better
last update pic

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -11,3 +11,4 @@
 @import "components/index";
 @import "pages/index";
 @import "pages/home";
+@import "pages/location"
```
Comment (chrononaut76) [inline app/assets/stylesheets/application.scss:14]: Missing semicolon

File: app/assets/stylesheets/pages/_location.scss
Comment (chrononaut76) [inline app/assets/stylesheets/pages/_location.scss:1]: CSS looks fine to me, but I'd also ask @gabrieleli09 or @DrewGodeke for feedback.

**Pull request merged**